### PR TITLE
`obs_mapping` and `var_mapping` only copy what is specified for `from_SCE`

### PR DESCRIPTION
**Pull request by LouiseDck**
Solves #265 but not yet #268

### CODE REVIEW

File: tests/testthat/test-SingleCellExperiment.R [hunk 1]
```diff
@@ -425,3 +425,61 @@ test_that("from_SCE works with unnamed mappings", {
...
+    skip_if(!is.null(msg), message = msg)
+
+    expect_true(var_key %in% colnames(ad_partial$var))
+    expect_equal(
+      ad_partial$var[[var_key]],
+      rowData(sce)[[var_from]],
+      info = paste0("var_key: ", var_key)
+    )
+  })
+}
```
Comment (rcannood) [inline tests/testthat/test-SingleCellExperiment.R:485]: can you also test whether the other structs are indeed empty?

**Review by rcannood — COMMENTED** [review]
minor request :)
